main 2021-07-02 13:43:46,634 INFO  [urjanet.think.update.UpdateRunner]: Platform version: 0.0
main 2021-07-02 13:43:46,636 INFO  [urjanet.think.update.UpdateRunner]: Running in portal-integration mode...
main 2021-07-02 13:43:46,643 INFO  [urjanet.environment.UrjanetProperties]: Loading UrjanetProperties - requestedEnv = dev, useConfigService = false
main 2021-07-02 13:43:46,643 INFO  [urjanet.environment.UrjanetProperties]: Loading properties from /opt/urjanet_configuration/trunk/urjanet.properties
main 2021-07-02 13:43:46,643 ERROR [urjanet.environment.UrjanetProperties]: Property file /opt/urjanet_configuration/trunk/urjanet.properties does not exist
main 2021-07-02 13:43:46,643 INFO  [urjanet.environment.UrjanetProperties]: Loading properties from /opt/urjanet_configuration/trunk/urjanet.properties.dev
main 2021-07-02 13:43:46,658 INFO  [urjanet.environment.UrjanetProperties]: env = null
main 2021-07-02 13:43:46,659 INFO  [urjanet.environment.UrjanetProperties]: environment.name = null
main 2021-07-02 13:43:46,659 INFO  [urjanet.environment.UrjanetProperties]: deploymentName = dev
main 2021-07-02 13:43:46,659 INFO  [urjanet.environment.UrjanetProperties]: deploymentEnv = dev
main 2021-07-02 13:43:46,659 INFO  [urjanet.conduit.message.MessageQueueAccessZ]: Initializing c2p message queue access from explicit properties file...environment = dev, c2p queue = null, s3 bucket = null
main 2021-07-02 13:43:46,659 INFO  [urjanet.conduit.message.MessageQueueAccessZ]: Initializing LocalQueueAccess for dev/local environment.
main 2021-07-02 13:43:46,680 DEBUG [urjanet.think.update.UpdateRunnerOptions]: Overriding acquisitionType to LOCAL_FILE
main 2021-07-02 13:43:46,680 DEBUG [urjanet.think.update.UpdateRunnerOptions]: Overriding sourceDirectory to /home/mooizza/Desktop/bills/
main 2021-07-02 13:43:46,687 DEBUG [urjanet.event.BaseEventManager]: Kinesis disabled - no single customer set
main 2021-07-02 13:43:46,773 DEBUG [urjanet.think.automon.publisher.AutomonEventPublisher]: Using queue : automon-messageQ-dev for Environment : dev
main 2021-07-02 13:43:46,780 TRACE [urjanet.store.EntityManagerFactories]: Initializing persistence unit: core
main 2021-07-02 13:44:49,127 ERROR [org.hibernate.tool.hbm2ddl.SchemaValidator]: could not get database metadata
java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:51)
	at org.hibernate.tool.hbm2ddl.SchemaValidator.validate(SchemaValidator.java:130)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:378)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at org.hibernate.ejb.HibernatePersistence.createEntityManagerFactory(HibernatePersistence.java:57)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:63)
	at urjanet.store.EntityManagerFactories.getFactory(EntityManagerFactories.java:91)
	at urjanet.store.ThreadLocalJpaTransactionManager.<init>(ThreadLocalJpaTransactionManager.java:30)
	at urjanet.store.StoreZ.getManager(StoreZ.java:70)
	at urjanet.store.StoreZ.getStoreForAtomicTrans(StoreZ.java:40)
	at urjanet.store.CoreStoreZ.getStoreForAtomicTrans(CoreStoreZ.java:38)
	at urjanet.think.update.api.AccessChannelAPISync.<init>(AccessChannelAPISync.java:55)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:743)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 18 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:44:49,128 ERROR [org.hibernate.tool.hbm2ddl.SchemaValidator]: could not complete schema validation
java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:51)
	at org.hibernate.tool.hbm2ddl.SchemaValidator.validate(SchemaValidator.java:130)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:378)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at org.hibernate.ejb.HibernatePersistence.createEntityManagerFactory(HibernatePersistence.java:57)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:63)
	at urjanet.store.EntityManagerFactories.getFactory(EntityManagerFactories.java:91)
	at urjanet.store.ThreadLocalJpaTransactionManager.<init>(ThreadLocalJpaTransactionManager.java:30)
	at urjanet.store.StoreZ.getManager(StoreZ.java:70)
	at urjanet.store.StoreZ.getStoreForAtomicTrans(StoreZ.java:40)
	at urjanet.store.CoreStoreZ.getStoreForAtomicTrans(CoreStoreZ.java:38)
	at urjanet.think.update.api.AccessChannelAPISync.<init>(AccessChannelAPISync.java:55)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:743)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 18 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:44:49,223 INFO  [urjanet.pull.api.PortalApiClient]: PortalAPI UserName: updaterunner_internal_user, EndPoint: https://api.urjanet.net
main 2021-07-02 13:44:51,430 DEBUG [urjanet.pull.api.PortalApiClient]: Extraction Channel Id :Altisource_13526-SoCalGCo_1
main 2021-07-02 13:44:51,873 DEBUG [urjanet.portal.store.PortalStoreZ]: Store user check || Current user: null || Passed user: system
main 2021-07-02 13:44:51,873 TRACE [urjanet.store.EntityManagerFactories]: Initializing persistence unit: portal
main 2021-07-02 13:45:52,772 ERROR [org.hibernate.tool.hbm2ddl.SchemaValidator]: could not get database metadata
java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:51)
	at org.hibernate.tool.hbm2ddl.SchemaValidator.validate(SchemaValidator.java:130)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:378)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at org.hibernate.ejb.HibernatePersistence.createEntityManagerFactory(HibernatePersistence.java:57)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:63)
	at urjanet.store.EntityManagerFactories.getFactory(EntityManagerFactories.java:91)
	at urjanet.store.ThreadLocalJpaTransactionManager.<init>(ThreadLocalJpaTransactionManager.java:30)
	at urjanet.store.StoreZ.getManager(StoreZ.java:70)
	at urjanet.store.StoreZ.getStoreForAtomicTrans(StoreZ.java:40)
	at urjanet.portal.store.PortalStoreZ.createNewPortalStore(PortalStoreZ.java:499)
	at urjanet.portal.store.PortalStoreZ.getStore(PortalStoreZ.java:265)
	at urjanet.portal.store.PortalStoreZ.getStoreForAtomicTrans(PortalStoreZ.java:149)
	at urjanet.pull.core.PullJobInfo.addBasicAccessChannelTags(PullJobInfo.java:123)
	at urjanet.pull.core.PullJobInfo.add(PullJobInfo.java:60)
	at urjanet.event.BaseEventManager.addInfoTag(BaseEventManager.java:506)
	at urjanet.think.update.api.AccessChannelAPISync.getPortalAccessChannelByIds(AccessChannelAPISync.java:107)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:766)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 23 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:45:52,773 ERROR [org.hibernate.tool.hbm2ddl.SchemaValidator]: could not complete schema validation
java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.tool.hbm2ddl.SuppliedConnectionProviderConnectionHelper.prepare(SuppliedConnectionProviderConnectionHelper.java:51)
	at org.hibernate.tool.hbm2ddl.SchemaValidator.validate(SchemaValidator.java:130)
	at org.hibernate.impl.SessionFactoryImpl.<init>(SessionFactoryImpl.java:378)
	at org.hibernate.cfg.Configuration.buildSessionFactory(Configuration.java:1872)
	at org.hibernate.ejb.Ejb3Configuration.buildEntityManagerFactory(Ejb3Configuration.java:906)
	at org.hibernate.ejb.HibernatePersistence.createEntityManagerFactory(HibernatePersistence.java:57)
	at javax.persistence.Persistence.createEntityManagerFactory(Persistence.java:63)
	at urjanet.store.EntityManagerFactories.getFactory(EntityManagerFactories.java:91)
	at urjanet.store.ThreadLocalJpaTransactionManager.<init>(ThreadLocalJpaTransactionManager.java:30)
	at urjanet.store.StoreZ.getManager(StoreZ.java:70)
	at urjanet.store.StoreZ.getStoreForAtomicTrans(StoreZ.java:40)
	at urjanet.portal.store.PortalStoreZ.createNewPortalStore(PortalStoreZ.java:499)
	at urjanet.portal.store.PortalStoreZ.getStore(PortalStoreZ.java:265)
	at urjanet.portal.store.PortalStoreZ.getStoreForAtomicTrans(PortalStoreZ.java:149)
	at urjanet.pull.core.PullJobInfo.addBasicAccessChannelTags(PullJobInfo.java:123)
	at urjanet.pull.core.PullJobInfo.add(PullJobInfo.java:60)
	at urjanet.event.BaseEventManager.addInfoTag(BaseEventManager.java:506)
	at urjanet.think.update.api.AccessChannelAPISync.getPortalAccessChannelByIds(AccessChannelAPISync.java:107)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:766)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 23 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:46:22,913 ERROR [org.hibernate.util.JDBCExceptionReporter]: Connections could not be acquired from the underlying database!
main 2021-07-02 13:46:22,915 ERROR [java.lang.reflect.InvocationHandler]: Exception invoking Transactional...
urjanet.transact.TransactionException: General TransactionException
	at urjanet.store.ThreadLocalJpaTransactionManager.startOrJoinTrans(ThreadLocalJpaTransactionManager.java:94)
	at urjanet.store.JpaStoreTransaction.startOrJoin(JpaStoreTransaction.java:62)
	at urjanet.transact.TransactionInvocationHandler.invoke(TransactionInvocationHandler.java:35)
	at com.sun.proxy.$Proxy46.query(Unknown Source)
	at urjanet.portal.store.PortalStore.query(PortalStore.java:236)
	at urjanet.portal.store.PortalUserDao.getFullyHydratedPortalUserByUsername(PortalUserDao.java:311)
	at urjanet.portal.store.PortalStoreZ.validateUser(PortalStoreZ.java:294)
	at urjanet.portal.store.PortalStoreZ.createNewPortalStore(PortalStoreZ.java:506)
	at urjanet.portal.store.PortalStoreZ.getStore(PortalStoreZ.java:265)
	at urjanet.portal.store.PortalStoreZ.getStoreForAtomicTrans(PortalStoreZ.java:149)
	at urjanet.pull.core.PullJobInfo.addBasicAccessChannelTags(PullJobInfo.java:123)
	at urjanet.pull.core.PullJobInfo.add(PullJobInfo.java:60)
	at urjanet.event.BaseEventManager.addInfoTag(BaseEventManager.java:506)
	at urjanet.think.update.api.AccessChannelAPISync.getPortalAccessChannelByIds(AccessChannelAPISync.java:107)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:766)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1389)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1317)
	at org.hibernate.ejb.AbstractEntityManagerImpl.throwPersistenceException(AbstractEntityManagerImpl.java:1399)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:63)
	at urjanet.store.ThreadLocalJpaTransactionManager.startOrJoinTrans(ThreadLocalJpaTransactionManager.java:91)
	... 16 common frames omitted
Caused by: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:140)
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:128)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)
	at org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:160)
	at org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:81)
	at org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1473)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:60)
	... 17 common frames omitted
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)
	... 22 common frames omitted
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 25 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:46:22,916 ERROR [urjanet.portal.store.PortalStoreZ]: createNewPortalStore error
urjanet.transact.TransactionException: General TransactionException
	at urjanet.transact.TransactionInvocationHandler.invoke(TransactionInvocationHandler.java:48)
	at com.sun.proxy.$Proxy46.query(Unknown Source)
	at urjanet.portal.store.PortalStore.query(PortalStore.java:236)
	at urjanet.portal.store.PortalUserDao.getFullyHydratedPortalUserByUsername(PortalUserDao.java:311)
	at urjanet.portal.store.PortalStoreZ.validateUser(PortalStoreZ.java:294)
	at urjanet.portal.store.PortalStoreZ.createNewPortalStore(PortalStoreZ.java:506)
	at urjanet.portal.store.PortalStoreZ.getStore(PortalStoreZ.java:265)
	at urjanet.portal.store.PortalStoreZ.getStoreForAtomicTrans(PortalStoreZ.java:149)
	at urjanet.pull.core.PullJobInfo.addBasicAccessChannelTags(PullJobInfo.java:123)
	at urjanet.pull.core.PullJobInfo.add(PullJobInfo.java:60)
	at urjanet.event.BaseEventManager.addInfoTag(BaseEventManager.java:506)
	at urjanet.think.update.api.AccessChannelAPISync.getPortalAccessChannelByIds(AccessChannelAPISync.java:107)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:766)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: urjanet.transact.TransactionException: General TransactionException
	at urjanet.store.ThreadLocalJpaTransactionManager.startOrJoinTrans(ThreadLocalJpaTransactionManager.java:94)
	at urjanet.store.JpaStoreTransaction.startOrJoin(JpaStoreTransaction.java:62)
	at urjanet.transact.TransactionInvocationHandler.invoke(TransactionInvocationHandler.java:35)
	... 14 common frames omitted
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1389)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1317)
	at org.hibernate.ejb.AbstractEntityManagerImpl.throwPersistenceException(AbstractEntityManagerImpl.java:1399)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:63)
	at urjanet.store.ThreadLocalJpaTransactionManager.startOrJoinTrans(ThreadLocalJpaTransactionManager.java:91)
	... 16 common frames omitted
Caused by: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:140)
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:128)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)
	at org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:160)
	at org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:81)
	at org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1473)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:60)
	... 17 common frames omitted
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)
	... 22 common frames omitted
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 25 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:46:22,919 ERROR [urjanet.pull.core.PullJobInfo]: Issues in gathering additional access channel information...
java.lang.RuntimeException: urjanet.transact.TransactionException: General TransactionException
	at urjanet.portal.store.PortalStoreZ.createNewPortalStore(PortalStoreZ.java:526)
	at urjanet.portal.store.PortalStoreZ.getStore(PortalStoreZ.java:265)
	at urjanet.portal.store.PortalStoreZ.getStoreForAtomicTrans(PortalStoreZ.java:149)
	at urjanet.pull.core.PullJobInfo.addBasicAccessChannelTags(PullJobInfo.java:123)
	at urjanet.pull.core.PullJobInfo.add(PullJobInfo.java:60)
	at urjanet.event.BaseEventManager.addInfoTag(BaseEventManager.java:506)
	at urjanet.think.update.api.AccessChannelAPISync.getPortalAccessChannelByIds(AccessChannelAPISync.java:107)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:766)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: urjanet.transact.TransactionException: General TransactionException
	at urjanet.transact.TransactionInvocationHandler.invoke(TransactionInvocationHandler.java:48)
	at com.sun.proxy.$Proxy46.query(Unknown Source)
	at urjanet.portal.store.PortalStore.query(PortalStore.java:236)
	at urjanet.portal.store.PortalUserDao.getFullyHydratedPortalUserByUsername(PortalUserDao.java:311)
	at urjanet.portal.store.PortalStoreZ.validateUser(PortalStoreZ.java:294)
	at urjanet.portal.store.PortalStoreZ.createNewPortalStore(PortalStoreZ.java:506)
	... 9 common frames omitted
Caused by: urjanet.transact.TransactionException: General TransactionException
	at urjanet.store.ThreadLocalJpaTransactionManager.startOrJoinTrans(ThreadLocalJpaTransactionManager.java:94)
	at urjanet.store.JpaStoreTransaction.startOrJoin(JpaStoreTransaction.java:62)
	at urjanet.transact.TransactionInvocationHandler.invoke(TransactionInvocationHandler.java:35)
	... 14 common frames omitted
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1389)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1317)
	at org.hibernate.ejb.AbstractEntityManagerImpl.throwPersistenceException(AbstractEntityManagerImpl.java:1399)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:63)
	at urjanet.store.ThreadLocalJpaTransactionManager.startOrJoinTrans(ThreadLocalJpaTransactionManager.java:91)
	... 16 common frames omitted
Caused by: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:140)
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:128)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)
	at org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:160)
	at org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:81)
	at org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1473)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:60)
	... 17 common frames omitted
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)
	... 22 common frames omitted
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 25 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
main 2021-07-02 13:46:22,919 DEBUG [urjanet.think.update.UpdateRunnerOptions]: begin access channel synchronization ***************
main 2021-07-02 13:46:22,919 DEBUG [urjanet.think.update.api.AccessChannelAPISync]: Starting dry runs of access channels
main 2021-07-02 13:46:52,970 ERROR [org.hibernate.util.JDBCExceptionReporter]: Connections could not be acquired from the underlying database!
main 2021-07-02 13:46:52,971 ERROR [urjanet.think.update.UpdateRunner]: Update failed - uncaught top level exception...will exit
urjanet.transact.TransactionException: General TransactionException
	at urjanet.store.ThreadLocalJpaTransactionManager.startTrans(ThreadLocalJpaTransactionManager.java:115)
	at urjanet.store.JpaStoreTransaction.start(JpaStoreTransaction.java:68)
	at urjanet.think.update.api.AccessChannelAPISync.dryRun(AccessChannelAPISync.java:155)
	at urjanet.think.update.api.AccessChannelAPISync.syncAccessChannels(AccessChannelAPISync.java:183)
	at urjanet.think.update.UpdateRunnerOptions.syncAccessChannels(UpdateRunnerOptions.java:778)
	at urjanet.think.update.UpdateRunnerOptions.<init>(UpdateRunnerOptions.java:147)
	at urjanet.think.update.UpdateRunner.main(UpdateRunner.java:1225)
Caused by: javax.persistence.PersistenceException: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1389)
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1317)
	at org.hibernate.ejb.AbstractEntityManagerImpl.throwPersistenceException(AbstractEntityManagerImpl.java:1399)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:63)
	at urjanet.store.ThreadLocalJpaTransactionManager.startTrans(ThreadLocalJpaTransactionManager.java:112)
	... 6 common frames omitted
Caused by: org.hibernate.exception.GenericJDBCException: Cannot open connection
	at org.hibernate.exception.SQLStateConverter.handledNonSpecificException(SQLStateConverter.java:140)
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:128)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:66)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:52)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:449)
	at org.hibernate.jdbc.ConnectionManager.getConnection(ConnectionManager.java:167)
	at org.hibernate.jdbc.JDBCContext.connection(JDBCContext.java:160)
	at org.hibernate.transaction.JDBCTransaction.begin(JDBCTransaction.java:81)
	at org.hibernate.impl.SessionImpl.beginTransaction(SessionImpl.java:1473)
	at org.hibernate.ejb.TransactionImpl.begin(TransactionImpl.java:60)
	... 7 common frames omitted
Caused by: java.sql.SQLException: Connections could not be acquired from the underlying database!
	at com.mchange.v2.sql.SqlUtils.toSQLException(SqlUtils.java:118)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:692)
	at com.mchange.v2.c3p0.impl.AbstractPoolBackedDataSource.getConnection(AbstractPoolBackedDataSource.java:140)
	at org.hibernate.connection.C3P0ConnectionProvider.getConnection(C3P0ConnectionProvider.java:78)
	at org.hibernate.jdbc.ConnectionManager.openConnection(ConnectionManager.java:446)
	... 12 common frames omitted
Caused by: com.mchange.v2.resourcepool.CannotAcquireResourceException: A ResourcePool could not acquire a resource from its primary factory or source.
	at com.mchange.v2.resourcepool.BasicResourcePool.awaitAvailable(BasicResourcePool.java:1469)
	at com.mchange.v2.resourcepool.BasicResourcePool.prelimCheckoutResource(BasicResourcePool.java:644)
	at com.mchange.v2.resourcepool.BasicResourcePool.checkoutResource(BasicResourcePool.java:554)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutAndMarkConnectionInUse(C3P0PooledConnectionPool.java:758)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool.checkoutPooledConnection(C3P0PooledConnectionPool.java:685)
	... 15 common frames omitted
Caused by: java.sql.SQLException: Access denied for user 'urja'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:998)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3835)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3771)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:870)
	at com.mysql.jdbc.MysqlIO.proceedHandshakeWithPluggableAuthentication(MysqlIO.java:1659)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1206)
	at com.mysql.jdbc.ConnectionImpl.coreConnect(ConnectionImpl.java:2239)
	at com.mysql.jdbc.ConnectionImpl.connectOneTryOnly(ConnectionImpl.java:2270)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2069)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:794)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:44)
	at sun.reflect.GeneratedConstructorAccessor26.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:389)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:399)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:325)
	at com.mchange.v2.c3p0.DriverManagerDataSource.getConnection(DriverManagerDataSource.java:175)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:220)
	at com.mchange.v2.c3p0.WrapperConnectionPoolDataSource.getPooledConnection(WrapperConnectionPoolDataSource.java:206)
	at com.mchange.v2.c3p0.impl.C3P0PooledConnectionPool$1PooledConnectionResourcePoolManager.acquireResource(C3P0PooledConnectionPool.java:203)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquire(BasicResourcePool.java:1138)
	at com.mchange.v2.resourcepool.BasicResourcePool.doAcquireAndDecrementPendingAcquiresWithinLockOnSuccess(BasicResourcePool.java:1125)
	at com.mchange.v2.resourcepool.BasicResourcePool.access$700(BasicResourcePool.java:44)
	at com.mchange.v2.resourcepool.BasicResourcePool$ScatteredAcquireTask.run(BasicResourcePool.java:1870)
	at com.mchange.v2.async.ThreadPoolAsynchronousRunner$PoolThread.run(ThreadPoolAsynchronousRunner.java:696)
